rpc/net.go

```diff
@@ -126,8 +136,14 @@ func (s *NetService) Discover(_ *http.Request, req *rpctypes.DiscoverRequest, re
 }
 
 // QueryPeer queries a peer for the coins they provide, their maximum amounts, and desired exchange rate.
-func (s *NetService) QueryPeer(_ *http.Request, req *rpctypes.QueryPeerRequest,
-	resp *rpctypes.QueryPeerResponse) error {
+func (s *NetService) QueryPeer(
+	_ *http.Request,
+	req *rpctypes.QueryPeerRequest,
+	resp *rpctypes.QueryPeerResponse,
+) error {
+	if s.isBootnode {
+		return errUnsupportedForBootnode
+	}
```

Comment:
This one too doesn't require a maker or a taker subsystem locally.

Reply:
same comment as queryAll above